### ISSUE TITLE
Add command line option for passing arguments on to pip

### ISSUE THREAD

**Issue by hhowe29** [pypa/pipenv#4643]
### Is your feature request related to a problem? Please describe.

I recently found myself in corporate network hell where I had to install packages through a proxy tunnel. `pip` allows me to specify a proxy and to force it to trust specific repo hosts. This enabled me to install with `pip`, but `pipenv` doesn't support these args.

Because of this, I had to create a venv with `pipenv --python 3.7`, and then `pip install` from inside a `pipenv shell` session. 

### Describe the solution you'd like

Please add a command line option for passing additional args onto pip. Something like 

```
pipenv sync --pip-args "--proxy socks5h://127.0.0.1:8029 --trusted-host pypi.org"
```

This option should be available on any command that relies on pip under the hood.

### Additional context

```
python 3.7.10
pipenv 2020.11.15
```

**Comment by NokiDev** [comment]
Hello @hhowe29 , I think you may be able to achieve what you want by using pip env variables. 
https://pip.pypa.io/en/stable/user_guide/#environment-variables 

I haven't tested this. 
But am looking for something similar regarding --global-options --debug, found your issue and this. 

In your case PIP_PROXY, and PIP_TRUSTED_HOST may do the job ?

**Comment by hhowe29** [comment]
@NokiDev Thanks for the info. That approach is good enough for my needs. I used it on a recent update of packages, and it worked without any problems.

pipenv devs, feel free to close.

**Comment by oz123** [comment]
Actually, this is fixed is pipenv version 2022.9.8:

See here https://github.com/pypa/pipenv/blob/main/CHANGELOG.rst#pipenv-202298-2022-09-08
